src/Microsoft.AspNet.Hosting/HostingServices.cs

```diff
@@ -69,21 +64,5 @@ public static IServiceCollection Create(IServiceProvider fallbackServices, Actio
             services.AddHosting(configuration);
             return services;
         }
-
-        // Manifest exposes the fallback manifest in addition to ITypeActivator, IHostingEnvironment, and ILoggerFactory
-        private class HostingManifest : IServiceManifest
```

Comment:
So I was thinking maybe just a static IDisposable HostingEngine.Start(HostingContext), and we move the fallback/configureServices into the context for now.